### DISCUSSION TITLE
Getting `Unable to parse input: "Parse error"` when running `bat <file>`

### DISCUSSION THREAD

**Discussion by TDH55** [sharkdp/bat#3106] [Q&A]
Hi, I've been using bat for a couple of years now and have recently started getting a `Unable to parse input: "Parse errror"` response when trying to use bat to view files. This seems to be happening with all file types, I've generated a new config file as well as uninstalling and reinstalling bat but haven't had any lucky finding a fix. Has anyone else had this happen or have any ideas what may be going on?

**Comment by keith-hall** [comment]
Hi, sorry for the slow response. I don't think I've seen or heard of this issue before. Does changing `BAT_CONFIG_PATH` or `BAT_CONFIG_DIR` help at all, i.e. get it to look at an non-existent config so it for sure uses the defaults?
When you reinstalled `bat`, how did you do so? Reinstalling from `cargo` or building from source could give different results from taking the latest released version, for example.
Does it work with `--paging=never` ? if so, it could be your `PAGER` causing problems...
